.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "trailingComma": "es5"
```

Comment:
❓ Why not keeping default prettier config also for that?

Reply:
I see! You right. I'm ok to keep it like this for now. I guess it can come with the change on not having override on prettier one day 😇